.dockerfile

```diff
@@ -1,17 +1,13 @@
 ARG HEMLOCK_PLATFORM=$BUILDPLATFORM
 ARG HEMLOCK_UBUNTU_TAG
 FROM --platform=${HEMLOCK_PLATFORM} ubuntu:${HEMLOCK_UBUNTU_TAG} AS base
-RUN --mount=type=cache,target=/var/cache/apt \
-    --mount=type=cache,target=/var/lib/apt \
-    rm /etc/apt/apt.conf.d/docker-clean \
```

Comment:
Can you educate me on why these mounts aren't needed anymore (here and below)?

Reply:
We never actually needed them. They allowed us to keep the download artifacts for apt packages (similar story for the removed opam packages) in a docker-managed volume. Basically, for this single command, apt would have access to the same downloaded artifacts as previous runs and might be able to use them without actually downloading.

We really don't need this, since these stages rarely change. We're not re-downloading over and over like I'd worried we would. Also, while making changes to the dockerfile for this PR, I ran into a lot of problems with permissions and lock state in the opam caches. The best solution I think is to just remove the complexity.

Reply:
Rewrapped the commit message at 80 columns.